src/Ledger/PDF.lagda

```diff
@@ -21,6 +23,12 @@
 \newunicodechar{≢}{\ensuremath{\nequiv}}
 \newunicodechar{❴}{\ensuremath{\{}}
 \newunicodechar{❵}{\ensuremath{\}}}
+\newunicodechar{⊢}{\ensuremath{\vdash}}
+\newunicodechar{⇀}{\ensuremath{\rightharpoonup}}
+-- TODO: replace the map arrow in the Agda code, and figure something out for the parentheses
```

Comment:
Is this still TODO?  I'm not sure what's left to do here.  Is there a ticket for this?

Reply:
Your PR does half of this, so I'll remove that part when merging it.

Reply:
The issue here is that one of the fonts doesn't like `⦇`, so I'm switching it to `(` here. Maybe we can do something better in the future.